migrations/20240217102522-create-settings.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      'Settings',
+      {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER,
+        },
+        name: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        data_type: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        account_id: {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+          references: {
+            model: 'Accounts',
+            key: 'id',
+          },
+        },
+        value: {
+          allowNull: false,
+          type: Sequelize.STRING,
+        },
+        createdAt: {
```

Comment:
nit: the case of the column names are inconsistent, let's use `snake_case` for all columns

Reply:
The google doc described that the fields for timestamps such as created_at should be camel case createdAt.

Reply:
Ok, I have Updated the doc. We used camel-cased column names in the past, but they are a big pain when writing raw SQL for querying the DB. So we've decided that moving forward we'll be sticking to `snake_cased` names for all columns.

Reply:
[Done](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/migrations/20240217102516-create-accounts.js#L18)